media-libs/nvidia-vaapi-driver/nvidia-vaapi-driver-0.0.13.ebuild

```diff
@@ -12,8 +12,9 @@ SRC_URI="https://github.com/elFarto/nvidia-vaapi-driver/archive/refs/tags/v${PV}
 LICENSE="MIT"
 SLOT="0"
 KEYWORDS="~amd64"
+IUSE="gst"
 
-RDEPEND="media-libs/gst-plugins-bad
+RDEPEND="gst? ( media-libs/gst-plugins-bad )
```

Comment:
This doesn't look right:
* The commit message says "gstream" (not "gstreamer")
* The USE flag is usually "gstreamer", not "gst"
* The dependency here is automagic if you're not adding a configure option to control it (i.e. if you have `USE=-gst` and `gst-plugins-bad` installed, won't it still be used)? Not withstanding that gst-plugins-bad is a plugin, one surely needs a dep on some part of gstreamer to build against if no build option is passed to change that.

Reply:
Real fix would be to add a proper meson option for it **in upstream**:
https://wiki.gentoo.org/wiki/Project:Quality_Assurance/Automagic_dependencies#Meson